src/common/dao/project.go

```diff
@@ -180,99 +181,102 @@ func SearchProjects(userID int) ([]models.Project, error) {
 	return projects, nil
 }
 
-//GetTotalOfUserRelevantProjects returns the total count of
-// user relevant projects
-func GetTotalOfUserRelevantProjects(userID int, projectName string) (int64, error) {
-	o := GetOrmer()
-	sql := `select count(*) from project p 
-			left join project_member pm 
-			on p.project_id = pm.project_id
-	 		where p.deleted = 0 and pm.user_id= ?`
-
-	queryParam := []interface{}{}
-	queryParam = append(queryParam, userID)
-	if projectName != "" {
-		sql += " and p.name like ? "
-		queryParam = append(queryParam, "%"+escape(projectName)+"%")
-	}
+// GetTotalOfProjects returns the total count of projects
+// according to the query conditions
+func GetTotalOfProjects(owner, name, public, member string,
+	role int) (int64, error) {
+	sql, params := queryConditions(owner, name, public, member, role)
 
-	var total int64
-	err := o.Raw(sql, queryParam).QueryRow(&total)
+	sql = `select count(*) ` + sql
 
+	var total int64
+	err := GetOrmer().Raw(sql, params).QueryRow(&total)
 	return total, err
 }
 
-// GetUserRelevantProjects returns the user relevant projects
-// args[0]: public, args[1]: limit, args[2]: offset
-func GetUserRelevantProjects(userID int, projectName string, args ...int64) ([]models.Project, error) {
-	return getProjects(userID, projectName, args...)
+// GetProjects returns a project list according to the query conditions
+func GetProjects(owner, name, public, member string,
+	role int, page, size int64) ([]*models.Project, error) {
```

Comment:
A little confuse about these two parameters -- page and size.  The swagger file gives the name are page and page_size, and also gives that the default value. But don't see any default value in the source code.